q2_feature_table/tests/test_examples.py

```diff
@@ -0,0 +1,16 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016-2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+from qiime2.plugin.testing import TestPluginBase
+
+
+class TestUsageExample(TestPluginBase):
+    package = 'q2_feature_table.tests'
+
+    def test_usage_example_ft_merge(self):
```

Comment:
```suggestion
    def test_usage_examples(self):
```

Reply:
Awesome, I made the change the function name as suggested. This PR will probably fail a test because of the overlap_method input. However, you mentioned #530 for the framework will adjust this problem, so we might be okay. Thanks.